recipes/comet-ms/build.sh

```diff
@@ -7,6 +7,7 @@ export CPPFLAGS="-I${PREFIX}/MSToolkit/include"
 
 sed -i.bak "s#gcc#${CC}#;s#g++#${CXX}#" MSToolkit/Makefile
 sed -i.bak "s#gcc#${CC}#;s#g++#${CXX}#" CometSearch/Makefile
+sed -i.bak "s#\$\(LIBS\)#\$\(LIBS\) -lrt#" Makefile
```

Comment:
```suggestion
sed -i.bak "s#\\$\(LIBS\)#\$\(LIBS\) -lrt#" Makefile
```

Maybe double escape because of bash?